programs/client/Client.cpp

```diff
@@ -524,6 +535,21 @@ class Client : public Poco::Util::Application
     {
         UseSSL use_ssl;
 
+        MainThreadStatus::getInstance();
+
+        if (config().has("max_memory_usage_in_client"))
+        {
+        /// Limit on total memory usage
+        size_t max_client_memory_usage = config().getInt64("max_memory_usage_in_client");
```

Comment:
@FArthur-cmd  consider to use config().getInt64("max_memory_usage_in_client", 0 /*default value*/);